sesoc.code-workspace

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
When editing, make sure to jump into this workspace configuration in order to get noice features like auto formatting when saving files!!